nodes/src/nodes/dataset_cobalt/IInstance.py

```
@@ -0,0 +1,92 @@
+            q = copy.deepcopy(question)
+
+            # Set the question text from the dataset item, replacing any
+            # prompt carried by the incoming template so emitted items
+            # contain only the dataset's prompt.
+            text = item.get('text', '')
+            if text:
+                if hasattr(q, 'questions'):
+                    try:
+                        q.questions = []
+                    except (ValueError, AttributeError):
+                        pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(ValueError, AttributeError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(ValueError, AttributeError): ...`

(SIM105)